components/autocomplete/Autocomplete.scss

```diff
@@ -0,0 +1,22 @@
+.autocomplete input {
```

Comment:
As a good practice, try to not use nesting, define a CSS class on the input. If the component evolves and includes and other `input` this selector will break it.

Reply:
For input I can, but for <ul> I don't have any control :(